estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,48 @@
+from odoo import api, fields, models
+from datetime import timedelta
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Offer Property Model"
+
+    price = fields.Float("Price")
+    status = fields.Selection(
+        selection=[
+            ('accepted', 'Accepted'),
+            ('rejected', 'Rejected')
+        ],
+        copy=False
+    )
+    partner_id = fields.Many2one("res.partner", required=True)
+    property_id = fields.Many2one("estate.property", required=True)
+    validity = fields.Integer("Validity", default=7)
+    date_deadline = fields.Date("Deadline", compute="_compute_deadline_date", inverse="_inverse_deadline_date")
```

Comment:
```suggestion
    date_deadline = fields.Date("Deadline", compute="_compute_deadline_date", inverse="_inverse_deadline_date")
```
what is the significance of `inverse`?

Reply:
The `inverse` parameter along with `compute` is used to allow the computed field to be editable and allow users to manually store the value(by default the compute fields are `readonly`).

`eg` in our model (`estate_property_offer`) the `date_deadline` is computed based on `validity`, but with the inverse method user can edit the `date_deadline` manually and it will update `validity` accordingly (user has to save the record first to see the updated value.)